### ISSUE TITLE
NTR: Cryptococcal antigen in CSF (Lab: cerebrospinal fluid tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8456]
New term request Cryptococcal antigen in CSF (samant_101029002405): Cerebrospinal fluid

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000761
- label: Positive CSF cryptococcal antigen test
- parents: [Positive CSF infectious agent test (HP:0430123)](https://hpo.jax.org/app/browse/term/HP:0430123)
- pmid: 
- def: A test result demonstrating the presence of a cryptococal antigen in the cerebrospinal fluid (CSF).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8456
- synonyms:
